OpenLIFUTransducerTracker/OpenLIFUTransducerTracker.py

```diff
@@ -206,24 +202,11 @@ def updateInputOptions(self):
 
         # Determine whether transducer tracking can be run based on the status of combo boxes
         self.checkCanRunTracking()
-            
-    def onLoadTransducerRegistrationSurfaceClicked(self):
-        qsettings = qt.QSettings()
-
-        filepath: str = qt.QFileDialog.getOpenFileName(
-            slicer.util.mainWindow(), # parent
-            'Load transducer', # title of dialog
-            qsettings.value('OpenLIFU/databaseDirectory','.'), # starting dir, with default of '.'
-            "Model (*.obj *.vtk);;All Files (*)", # file type filter
-        )
-        if filepath:
-            self.activeTRS = slicer.util.loadModel(filepath) # Temporary approach
-            self.checkCanRunTracking()
 
     def checkCanRunTracking(self,caller = None, event = None) -> None:
```

Comment:
In this function do we now also need to check that the loaded transducer has a registration surface?
Since we allow transducers to be loaded that do not have an affiliated surface model.

If the surface model is not present and that's the only reason the tracking button is disabled, then the runTrackingButton tooltip could be "This transducer does not have an affiliated registration surface, which is needed to run tracking."